content/rdi/rdi-faq.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: F.A.Q.
+linkTitle: FAQ
+description: Questions and answers about RDI
+weight: 70
+alwaysopen: false
+categories: ["redis-di"]
+aliases:
+---
+
+## General
+
+**WHich use cases are supported by RDI?**
+RDI supports the Ingest and Write behind use cases.
+Ingest is used to keep Redis in sync with the application system of records database so that read queries can be done in speed and scale from Redis.
+Write behind is used to speed up application read and writes using Redis as front database while asynchronously updating a downstream database.
```

Comment:
```suggestion
**Which use cases are supported by RDI?**
RDI supports two use cases: ingest and write-behind.
Ingest is used to keep Redis in sync with the application system of record database so that read queries can be done in speed and at scale from Redis.
Write behind is used to speed up application reads and writes using Redis as the front-end database while asynchronously updating a downstream database.
```

Reply:
done